### DISCUSSION TITLE
Can this be used on non SD User images for Upscaling?

### DISCUSSION THREAD

**Discussion by disneytoy** [pkuliyi2015/multidiffusion-upscaler-for-automatic1111#355] [Q&A]
Greetings,

Just watched a youtube on your project, very impressive!  Can I use you upscaler on my own images? (img2img)?

Here's my problem,  I use MidJourney. Their 2x upscaler is not very good. Images are great. I have been bringing my 2048 pixel images into KREA.AI Enhancer/upscaler. Limited to 2x (free version), so I get the same resolution output, but it does an amazing job of enhancing the detail. I print these and it does make a big difference even with no upscale. They offer 4x and 8x but at a Pro price.

I'm pretty sure the two guys in Florida behind Krea.ai are just using SD. I could not find out what upscaler they were using. Anyone know?

So if I have my 2048 images, would I be able to use your upscaler as img2img?

Thanks

Great work.

Max

**Comment by nivedwho** [comment]
@disneytoy did you find some solution to this?
